src/Command/NotifyRemove.js

```diff
@@ -0,0 +1,51 @@
+const Command = require('../Command');
+
+const messages = {
+    noGame: 'You must specify one of the available notifications to unsubscribe from: {types}',
+    notifyGameSeparator: 'NOTIFY_GAME_SEPARATOR',
+    cantRemove: 'I was unable to remove the role from your user.',
+    success: 'Your role has been removed.'
+};
+
+module.exports = Command.extend({
+    config: null,
+    shouldDeleteMessage: true,
+    commandName: 'notifyremove',
+    commandAliases: ['unwar'],
+    dependencies: {
+        'config': 'config'
+    },
+    initialize: function (dependencyGraph) {
+        this._super(dependencyGraph);
+    },
+    processMessage: async function (message, tokens, commandName) {
+        let notifyGame;
+        if (this.commandAliases.indexOf(commandName) >= 0) {
+            notifyGame = 'warband';
+        } else if (tokens.length < 2) {
+            return message.channel.send(this.i18n.__mf(
+                messages.noGame,
+                {
+                    types: Object.keys(this.config.notificationRoles).join(this.i18n.__mf(messages.notifyGameSeparator))
+                }
+            ));
+        } else {
+            notifyGame = tokens[1].toLowerCase();
+        }
+
+        const role = message.guild.roles.cache.get(this.config.notificationRoles[notifyGame]);
```

Comment:
There's no validation that `notifyGame` is in the array, so `role` can end up undefined before being passed into `.cache.get`.  Unsure what will happen at that point. Unlikely to be anything good.

Reply:
I tested this, nothing will happen — if they don't already have the role, it will just continue as normal with no effect.

Reply:
You misunderstand. I mean if a game role that doesn't exist is specified